docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -47,6 +47,50 @@ Include the following properties in your `pom.xml` file to track the version of
 ----
 --
 
+. Specify the repositories containing the {SpringBoot} Maven plugin and the {SpringBoot} dependencies you are using in your application.
```

Comment:
Please capitalise both words in _Maven Plugin._ [consistency with the doc set]

Also, please end the sentence with a colon. [stylistics]

Reply:
Will do, on both points.